packages/atomic/cypress/integration/result-list/result-list.cypress.ts

```diff
@@ -103,6 +103,18 @@ describe('Result List Component', () => {
           .find(lineHeightSelector)
           .should('be.visible');
       });
+
+      it(`should pass the display-${display} class to the list root`, () => {
```

Comment:
Originally, I didn't add these tests because I thought that to mess this up it would require a very stupid mistake, which it does, but somehow I made that mistake. Lesson learned 😅